src/08-leds-again/auxiliary/src/lib.rs

```diff
@@ -33,19 +35,25 @@ fn panic(info: &PanicInfo) -> ! {
 
     cortex_m::asm::bkpt();
 
-    loop {}
+    loop {
+        atomic::compiler_fence(Ordering::SeqCst)
```

Comment:
do these compiler fences affect the output binary size at all? I know they won't insert any code directly, but do they change the code generation by LLVM at all?

Reply:
Gotcha. I didn't know if this would prevent LLVM from optimizing something (causing a larger binary). Thanks for the clarification.

Reply:
Actually. Do you explain what these are/why they are necessary anywhere?

Reply:
In the commit message, but I can add a comment like [this one](https://github.com/rust-embedded/cortex-m-rt/blob/795c86c296b36fdca75a742ab20ea678d71039af/src/lib.rs#L534) to each loop.

Reply:
I think just on the first instance would be fine IMO, or even in the text itself.

Reply:
This code is not user facing. It's part of the auxiliary crate that expose some API that the reader will use, not part of the code they'll modify. 

I'll add a comment to each file since curious readers may look at the code.